Dockerfile.operator

```
@@ -37,12 +37,25 @@ RUN exportOrFail() { echo $1; if [[ $1 == *= ]]; then echo "Error: empty variabl
               -X ${VERSION_PKG}.autoInstrumentationApacheHttpd=${AUTO_INSTRUMENTATION_APACHE_HTTPD_VERSION} \
               -X ${VERSION_PKG}.autoInstrumentationNginx=${AUTO_INSTRUMENTATION_NGINX_VERSION}"
 
-FROM registry.redhat.io/ubi8/ubi-minimal:latest@sha256:33161cf5ec11ea13bfe60cad64f56a3aa4d893852e8ec44b2fd2a6b40cc38539
+
+
+
+FROM registry.redhat.io/ubi8/ubi-micro:latest@sha256:eae27ba458e682d6d830f6c77c9e3a4c33cf1718461397b741e674d9d37450f3 AS target-base
+FROM registry.redhat.io/ubi8/ubi:latest@sha256:8bd1b6306f8164de7fb0974031a0f903bd3ab3e6bcab835854d3d9a1a74ea5db as ubi-micro-additional-packages
+COPY --from=target-base / /mnt/rootfs
+# RUN rpm --root /mnt/rootfs --import /etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release
+# --releasever 8 --setopt install_weak_deps=false --nodocs
+RUN dnf update -y && rm -rf /var/cache/yum && \
```

`dnf update`/`dnf upgrade` is not supported in hermetic mode.
```suggestion
RUN rm -rf /var/cache/yum && \
```
Also, you may want to move `rm -rf /var/cache/yum` to the point after `dnf clean all`.